src/app/providers/index.ts

```diff
@@ -0,0 +1,8 @@
+import compose from 'compose-function';
+import { withAlert } from './with-alert';
+import { withI18Next } from './with-i18next';
+import { withRouter } from './with-router';
+import { withTheme } from './with-theme';
+import './with-firebase';
```

Comment:
is there any neccessity for importing `firebase` here?

Reply:
Actually, it's not necessary, but I thought it would be clearer if we gather all the providers in the root provider